syntax = "proto3";

package user_token;

message UserToken {
    string id = 1;
    optional string user_id = 2;
    string token = 3;
    string created_at = 4;
    string expired_at = 5;
}